src/components/auth/torus/AuthTorus.web.js

```diff
@@ -190,37 +237,34 @@ const AuthTorus = ({ screenProps, navigation, styles, store }) => {
         const { source } = await ready(replacing)
 
         log.debug('showing checkmark dialog')
+
         // showLoadingDialog(true)
         // await delay(30000000)
 
         // await new Promise(res => showLoadingDialog(true, res))
         // showLoadingDialog(true)
         // log.debug('hiding checkmark dialog')
         hideDialog()
-
-        //user exists reload with dashboard route
+        if (isSignUp) {
+          if (exists) {
+            return showEmailUsedDialog(provider, source)
+          }
+          fireEvent(SIGNUP_STARTED, { source, provider })
+          return navigate('Signup', {
+            regMethod: REGISTRATION_METHOD_TORUS,
+            torusUser,
+            torusProvider: provider,
+          })
+        }
         if (exists) {
+          successDialog()
```

Comment:
this is not the correct place to start showing the dialog. and its not a success dialog.
are you working with eslint enabled? because there are unused functions here